cddl/usr.sbin/zfsd/case_file.h

```diff
@@ -235,18 +235,27 @@ class CaseFile
 	 */
 	int IsSpare();
 
+	/*
+	 * \brief Get case vdev's specified property
+	 */
+	int GetVdevProp(vdev_prop_t) const;
+
 protected:
 	enum {
+		/**
```

Comment:
Actually, my previous suggestion was slightly wrong.  Yes, you should've had a blank line here.  But it should only get one * instead of two, because it isn't a Doxygen comment for the next item.
```suggestion
		/*
```

Reply:
ok. Also I think I should have two * for GetVdevProps() though